ui/component/model/src/index.ts

```diff
@@ -1,2 +1,2521 @@
-export * from "./model";
-export {AssetModelUtil} from "./util";
```

Comment:
Not sure how this has happened but the autogenerated contents get output to `model.ts` but they seem to have been copied into here?

Reply:
MMMh I do not intend myself to copy this content. Don't really know what could have create such case.

Reply:
I don't really know which action lead to that change but I fix this by reverting the changes